packages/components/components/elvis-datepicker-range/package.json

```
@@ -20,7 +20,7 @@
     "@elvia/elvis-component-wrapper": "^4.2.0",
     "@elvia/elvis-datepicker": "^9.0.12",
```

Like this:
```suggestion
    "@elvia/elvis-datepicker": "^9.1.0",
```
Remember to run `yarn` when updating this PR